lib/tinytex.cpp

```diff
@@ -11,10 +13,32 @@ volatile bool TinyTeX::_isInited = false;
 std::string TinyTeX::_defaultMathFontName;
 std::string TinyTeX::_defaultMainFontName;
 
-void TinyTeX::init(const FontSrc& mathFontSrc) {
-  FontContext::addMathFont(mathFontSrc);
-  _defaultMathFontName = mathFontSrc.name;
-  if (_isInited) return;
+void TinyTeX::init(Init init) {
+  const FontSrc** mathFontSrc = std::get_if<const FontSrc*>(&init);
+  if (mathFontSrc) {
+    FontContext::addMathFont(**mathFontSrc);
+    _defaultMathFontName = (*mathFontSrc)->name;
+    goto initialization;
```

Comment:
I believe directly using `if (_isInited) return;` is better.

Reply:
the initialization is not just `if (_isInited) return;`, but also includes
```cpp
  initialization:if (_isInited) return;
  _isInited = true;
  NewCommandMacro::_init_();
```

Reply:
I've missed these lines... maybe lambda is a better choice?

```C++
auto initialization = [&]() {
  if (_isInited) return;
  _isInited = true;
  NewCommandMacro::_init_();
}
```